docs/csharp/whats-new/version-update-considerations.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Version and update considerations for C# developers
+description: Introducing new languages features in your library can impact the code that uses it.
+ms.date: 09/19/2018
+---
+
+# Version and update considerations for C# developers
+
+Compatibility is a very important goal as new features are added to the C# language. In almost all cases, you can use a new compiler version without breaking existing code.
+
+More care may be required when you adopt new language features in a library, and consumers of that library have not yet upgraded. The are two variations of compatibility: source compatible and binary compatible.
+
+## Source compatible changes
+
+Changes to your library are **source compatible** when assemblies that use your library must be recompiled against the new version to work correctly. Source code changes to the dependent library aren't required, but a rebuild is.
```

Comment:
Nit: "assemblies ... must be recompiled": is it a wide-spread usage? Or "assembly source code ... must be recompiled to produce a new assembly version"?

Reply:
This comment also helped me re-write the descriptions to reverse the order.  😃